lighthouse-core/config/v2/default.json

```diff
@@ -0,0 +1,308 @@
+{
+  "version": 2,
+
+  "gatherers": {
+    "url": {},
+    "https": {},
+    "viewport": {},
+    "theme-color": {},
+    "manifest": {},
+    "accessibility": {},
+    "image-usage": {},
+    "content-width": {},
+    "service-worker": {},
+    "offline": {},
+    "http-redirect": {},
+    "html-without-javascript": {},
+    "chrome-console-messages": {},
+    "styles": {},
+    "css-usage": {},
+    "all-event-listeners": {
+      "path": "dobetterweb/all-event-listeners"
+    },
+    "anchors-with-no-rel-noopener": {
+      "path": "dobetterweb/anchors-with-no-rel-noopener"
+    },
+    "appcache": {
+      "path": "dobetterweb/appcache"
+    },
+    "console-time-usage": {
+      "path": "dobetterweb/console-time-usage"
+    },
+    "datenow": {
+      "path": "dobetterweb/datenow"
+    },
+    "document-write": {
+      "path": "dobetterweb/document-write"
+    },
+    "geolocation-on-start": {
+      "path": "dobetterweb/geolocation-on-start"
+    },
+    "notification-on-start": {
+      "path": "dobetterweb/notification-on-start"
+    },
+    "domstats": {
+      "path": "dobetterweb/domstats"
+    },
+    "optimized-images": {
+      "path": "dobetterweb/optimized-images"
+    },
+    "tags-blocking-first-paint": {
+      "path": "dobetterweb/tags-blocking-first-paint"
+    },
+    "websql": {
+      "path": "dobetterweb/websql"
+    }
+  },
+
+  "passes": {
+    "defaultPass": {
+      "order": 10,
```

Comment:
can we kick `order` from the MVP so we can discuss it in a separate PR? I'm still not really a fan and with no motivating code that needs it yet there's no rush